tests/integration/s2n_handshake_test_s_client.py

```diff
@@ -632,6 +632,8 @@ def elliptic_curve_test(host, port, libcrypto_version, fips_mode, **kwargs):
     for noise.
     """
     supported_curves = ["P-256", "P-384"]
+    if not fips_mode:
```

Comment:
I'm only seeing "not fips" in the old integration tests, not in the new integration tests, source, or unit tests. What's the relationship between P-521 and fips? Am I missing the checks elsewhere?

Reply:
P-521 (unlike several other NIST curves) isn't for use in FIPS mode (so far as I can tell).

Reply:
How are we enforcing that in the source?

Reply:
I believe that P-521 isn't on the FIPS policies. But as far as a test to ensure that, I haven't checked.